src/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/ArtworkDetails.tsx

```diff
@@ -10,31 +10,43 @@ import { useRouter } from "System/Router/useRouter"
 import { artworkDetailsValidationSchema, validate } from "../Utils/validation"
 import { BackLink } from "Components/Links/BackLink"
 import { useSystemContext } from "System"
-import { createOrUpdateConsignSubmission } from "../Utils/createOrUpdateConsignSubmission"
+import {
+  createOrUpdateConsignSubmission,
+  SubmissionInput,
+} from "../Utils/createOrUpdateConsignSubmission"
 import { createFragmentContainer, graphql } from "react-relay"
 import {
   ArtworkDetails_submission,
   ConsignmentAttributionClass,
 } from "__generated__/ArtworkDetails_submission.graphql"
+import { ArtworkDetails_myCollectionArtworkSubmissionDetails } from "__generated__/ArtworkDetails_myCollectionArtworkSubmissionDetails.graphql"
 import { UtmParams } from "../Utils/types"
 import { getENV } from "Utils/getENV"
 import createLogger from "Utils/logger"
+import { ConsignmentSubmissionSource } from "__generated__/CreateConsignSubmissionMutation.graphql"
 
 const logger = createLogger("SubmissionFlow/ArtworkDetails.tsx")
 
 export interface ArtworkDetailsProps {
   submission?: ArtworkDetails_submission
+  myCollectionArtworkSubmissionDetails?: ArtworkDetails_myCollectionArtworkSubmissionDetails
```

Comment:
**suggestion (non-blocking):** Reading this it wasn't immediately clear to me what we're passing to this component. What do you think about renaming the variable name and the fragment to `myCollectionArtwork` to make clear that this object is an artwork (it comes from the GraphQL `Artwork` type). 

```suggestion
  myCollectionArtwork?: ArtworkDetails_myCollectionArtwork
```

Reply:
I think it'd be better :)